salesagility/SuiteCRM
# Fixed #2144 - SuiteP - Report PDF file - names of columns are missing

## Description



When exporting a report to PDF in suiteP the column headers matched the background color, making them invisible. This fix adjusts the color to allow the headers to be seen.

![pdf-img](https://user-images.githubusercontent.com/26431166/26870669-9fb0a8d8-4b68-11e7-9895-01a36dfe0a12.png)

Issue reference: #2144  
Issue reference: #3666

## Motivation and Context

This fix allows column headers to be visible in the suiteP theme when exporting PDFs.

## How To Test This

1. Create a report.
2. Download the report as pdf.
3. Check that column headers are visible.

## Types of changes


### Final checklist
